app/components/CustomTooltip/index.tsx

```diff
@@ -28,11 +38,69 @@ function CustomTooltip(props: Props) {
         value,
         minValue,
         maxValue,
+        customTooltipData,
     } = props;
 
-    const uncertaintyRange = format === 'percent'
-        ? ` [${minValue}% - ${maxValue}%]`
-        : ` [${formatNumber(format, minValue)} - ${formatNumber(format, maxValue)}]`;
+    const uncertaintyRange = useMemo(() => (
+        format === 'percent'
+            ? ` [${minValue}% - ${maxValue}%]`
+            : ` [${formatNumber(format, minValue)} - ${formatNumber(format, maxValue)}]`
+    ), [
+        minValue,
+        maxValue,
+        format,
+    ]);
+
+    const calculatedTotal = useMemo(() => (
+        customTooltipData?.reduce(
+            (acc, obj) => (acc + (obj?.contextIndicatorValue ?? 1)), 0,
+        ) ?? 1
+    ), [
+        customTooltipData,
```

Comment:
What does this do? Why is the default value `1`? Why not use an existing function?